tests/runtime/probe

```diff
@@ -256,6 +256,11 @@ PROG interval:s:1 { exit(); } END { printf("end"); }
 EXPECT end
 TIMEOUT 2
 
+NAME BEGIN,END
+PROG BEGIN,END { printf("Hello\n"); exit(); }
+EXPECT Hello
```

Comment:
Should there be two "Hello"s here? Can this test framework handle that?

Reply:
Good catch, it can, I updated the test.